src/common/lib/client/auth.ts

```diff
@@ -887,7 +810,7 @@ class Auth {
     }
   }
 
-  _saveTokenOptions(tokenParams: API.TokenParams | null, authOptions: API.AuthOptions | null) {
+  _saveTokenOptions(tokenParams: API.TokenParams | null, authOptions: API.AuthOptions | undefined | null) {
```

Comment:
can we mark `authOptions` param as optional here and in `_forceNewToken` instead of having to do `| undefined`?
the same would go for `tokenParams`.

Reply:
Have changed the call to `_forceNewToken` inside `authorize` so that it no longer requires a change of signature for `_saveTokenOptions` or `_forceNewToken`.

Reply:
cool!